### PR TITLE
Add workflow dispatch trigger for parent/child with cloud integration smoke tests

### PR DESCRIPTION
**Pull request by dimko**
##### Summary

- Update soon to be deprecated output variables to new format using name=<varialbe name> >> $GITHUB_OUTPUT
- Add an extra step to trigger agent smoke tests and wait for its completion before reporting workflow's status

##### Additional Information

Differences between the 2 triggers.

- Full regression triggers a number of tests that target staging environment, running on self hosted runners inside testing cluster. Full regression consists of both UI and API tests verifying multiple functionalities of the cloud app using a custom image of a netdata agent built from latest master. **This will soon be deprecated.**

- Agent smoke tests run API only tests, against production cloud application and use a custom image of netdata in a similar manner as Full Regression. This workflow runs on github runners and it's not affected by the downtime of the testing cluster as the Full Regression.

### CODE REVIEW

**Review by Ferroin — APPROVED** [review]

File: .github/workflows/cloud_regression.yml [hunk 1]
```diff
@@ -52,3 +52,18 @@ jobs:
...
+          ref: refs/heads/master
+          workflow: agent_smoke_tests.yml
+          token: ${{ secrets.NETDATABOT_GITHUB_TOKEN  }}
+          inputs: '{ "netdata_branch": "${{ steps.output-workflow-dispatch-params.outputs.netdata_branch }}",
+               "netdata_repo": "${{ steps.output-workflow-dispatch-params.outputs.netdata_repo }}",
+               "netdata_pr_number": "${{ steps.output-workflow-dispatch-params.outputs.netdata_pr_number }}",
+               "netdata_branch_commit_hash": "${{ steps.output-workflow-dispatch-params.outputs.netdata_commit_hash }}",
+               "custom_netdata_image": "true"
+               }'
+          wait-for-completion: true
```
Comment (thiagoftsm) [inline .github/workflows/cloud_regression.yml:69]: Should we add a blank line at the end @Ferroin ?

Comment (Ferroin) [reply .github/workflows/cloud_regression.yml:69]: In theory yes, but in practice it really doesn’t matter much here.

Comment (dimko) [reply .github/workflows/cloud_regression.yml:69]: Isn't there one? My IDE shows it as having one at the end 🤔

Comment (Ferroin) [reply .github/workflows/cloud_regression.yml:69]: Both yamllint and the diff here on GitHub do not show one.

Comment (thiagoftsm) [reply .github/workflows/cloud_regression.yml:69]: @dimko , please , add one for us.

Comment (dimko) [reply .github/workflows/cloud_regression.yml:69]: @thiagoftsm 50$ 😛

Comment (thiagoftsm) [reply .github/workflows/cloud_regression.yml]: I will consider that you are asking for Uruguayan pesos :smile:

**Review by thiagoftsm — APPROVED** [review]

**Review by vkalintiris — APPROVED** [review]

**Pull request merged**